hibernate-core/src/test/java/org/hibernate/orm/test/multitenancy/schema/TenantResolverConfigurationTest.java

```diff
@@ -35,7 +35,7 @@ public void testConfiguration() throws Exception {
 	}
 
 	private static class TestCurrentTenantIdentifierResolver implements CurrentTenantIdentifierResolver {
-		private String currentTenantIdentifier;
+		private String currentTenantIdentifier = "acme";
```

Comment:
That seems irrelevant.

Reply:
That's actually a design question we have to ask ourselves. I think that right now, we allow null return values from the `resolveCurrentTenantIdentifier` method. Anyway, there are other implementations that also potentially return null, so either we fix all of them or none :)
Fixing the implementation could also mean checking for a null value and throwing an exception btw, which I would consider to be more correct.

Reply:
Please revert the change